htlcswitch/link.go

```diff
@@ -153,6 +153,14 @@ type ChannelLinkConfig struct {
 	// reestablishment message to the remote peer. It should be done if our
 	// clients have been restarted, or remote peer have been reconnected.
 	SyncStates bool
+
+	// BatchTimerInterval is the time between every time we will try to
```

Comment:
Suggested comment re-write: 
```
BatchTimerInterval is the interval that we'll use to check the batch to see if there're any updates we should flush out. By batching updates into a single commit, we attempt to increase throughput by maximizing the number of updates coalesced into a single commit. 
```

Reply:
Alternatively, this could instead be a `chan <- time.Time`. This may give testing code a greater degree of control, as it'll be able to instantaneously trigger an update check whenever it pleases.